sgt-back-end/index.js

```diff
@@ -0,0 +1,158 @@
+const pg = require('pg');
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const express = require('express');
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select "gradeId",
+           "name",
+           "course",
+           "score",
+           "createdAt"
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      if (!grades) {
+        res.status(404).json({
+          error: 'Cannot find grades'
+        });
+      } else {
+        res.json(grades);
+      }
+    })
```

Comment:
Even if the list of grades is empty, the server should still respond with a `200 OK` containing an empty array. `404` errors indicate the requested resource (`/api/grades`) doesn't exist. When that resource returns a list, the resource still exists even if the list is empty.

Reply:
Please address this issue sending a `200 OK` whether or not the list of grades is empty.